internal/handshake/token_generator.go

```diff
@@ -35,6 +36,7 @@ type token struct {
 	IsRetryToken             bool
 	RemoteAddr               []byte
 	Timestamp                int64
+	RTT                      int64 // in ms
```

Comment:
Millisecond is too low-resolution for some circumstances (e.g. data center traffic). In the session ticket, we used microsecond resolution. Any reason to change it?